deploy/template.yaml

```diff
@@ -1125,6 +1137,175 @@ Resources:
       DatapointsToAlarm: 1
       Threshold: 1
       ComparisonOperator: GreaterThanOrEqualToThreshold
+  
+  HmrcTokenFunction:
+    Type: AWS::Serverless::Function
+    DependsOn:
+      - "HmrcTokenLogGroup"
+    Properties:
+      FunctionName: !Sub "HmrcToken-${AWS::StackName}"
+      CodeUri: ../src/
+      Handler: HmrcTokenHandler.lambdaHandler
+      ReservedConcurrentExecutions: !If
+        - ApplyReservedConcurrency
+        - !Ref LambdaConcurrency
+        - !Ref AWS::NoValue
+      # Events:        
```

Comment:
Guessing this is no longer needed so can we remove please?

Reply:
Yes, plan is to remove this. I just want to get this working on main BE stack before i confirm and remove these commented unnecessary code.

Reply:
Removed as we have tested this in main stack and its working